libraries/Microsoft.Bot.Builder.AI.QnA/Models/RankerTypes.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Enumeration of types of ranking.
+    /// </summary>
+    public enum RankerTypes
```

Comment:
>RankerTypes [](start = 16, length = 11)

should have jsonproperty attribute
Should have JsonConverter(StringEnumConverter) attribte

Reply:
Modifying this to const string as Chris suggested. This will keep parity with NodeJS also,